presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/SqlServerQueryRunner.java

```diff
@@ -106,6 +112,7 @@ public static void main(String[] args)
 
         DistributedQueryRunner queryRunner = (DistributedQueryRunner) createSqlServerQueryRunner(
                 testingSqlServer,
+                ImmutableMap.of("http-server.http.port", "8080"),
```

Comment:
please add a commit message why it's needed

Reply:
we do so for virtually every oother QueryRunner.main.

should i say that ?

Reply:
added soimething

Reply:
Isn't 8080 the default?

Reply:
In Presto -- yes

in DistributedQueryRunner -- no